packages/sdk/utils/src/io.ts

```diff
@@ -98,98 +81,3 @@ export const AttestationServiceStatusResponseType = t.type({
   maxRerequestMins: t.number,
   twilioVerifySidProvided: t.boolean,
 })
```

Comment:
Does the comment apply to this too? Or do we still need `AttestationServiceStatusResponseType`?

Reply:
At one point I thought moving AttestationServiceStatusResponseType would be beneficial so i marked it as deprecated here. but it turns out I was wrong. So brought it back